crates/tests-e2e/src/setup.rs

```diff
@@ -57,14 +57,20 @@ impl TestSetup {
         }
     }
 
-    pub async fn fund_coordinator(&self, amount: Amount) {
+    /// Funds the coordinator with [`amount`/`in_utxos`] utxos
+    ///
+    /// E.g. if amount = 3 BTC, and in_utxos = 3, it would create 3 UTXOs a 1 BTC
+    pub async fn fund_coordinator(&self, amount: Amount, in_utxos: u64) {
         // Ensure that the coordinator has a free UTXO available.
         let address = self.coordinator.get_new_address().await.unwrap();
 
-        self.bitcoind
-            .send_to_address(&address, amount)
-            .await
-            .unwrap();
+        let sats_per_fund = amount.to_sat() / in_utxos;
+        for _ in 0..in_utxos {
+            self.bitcoind
+                .send_to_address(&address, Amount::from_sat(sats_per_fund))
```

Comment:
🙃 Almost certainly unnecessary, but you could use my recently introduced `send_multiple_utxos_to_address`.

With your approach, I think in testing I ran into a problem where I was generating too many transactions, so they wouldn't all fit in a single block!

Reply:
🙃 Can you document in the commit message why this change is needed? (A comment here is okay too.)

Reply:
the `reject_offer` test failed because we didn't have enough UTXOs on the second order. I first generated an other block in between the test (before opening the second position) but figured this might still be flaky - or we might need an other busy wait there. Hence, I changed the setup so that one can allow for multiple UTXOs at the getgo. I'll update the commit message